clients/web/src/models/UserModel.js

```diff
@@ -1,6 +1,26 @@
 girder.models.UserModel = girder.Model.extend({
     resourceName: 'user',
 
+    // Sets this model to the currently logged in user; has no effect is no user
+    // is logged in.
+    current: function (success, error) {
+        girder.restRequest({
+            method: 'GET',
+            path: '/user/me'
+        }).then(_.bind(function (user) {
+            if (user) {
+                this.set(user);
+                if (success) {
+                    success(this);
+                }
+            } else {
+                if (error) {
+                    error(this);
+                }
```

Comment:
I'm fine with either of those solutions. If we go with 1, let's just document that in the docstring and maybe also mention `girder.fetchCurrentUser` in the docstring.